### ISSUE TITLE
Add "-H windowsgui" to ldflags on windows?

### ISSUE THREAD

**Issue by emersion** [andlabs/ui#153]
Without this flag, a cmd window is opened with the app. Should we add this by default?

**Comment by F0rth** [comment]
yes!

**Comment by andlabs** [comment]
`#cgo` directives don't accept `go build` flags. Is there some other way?

**Comment by F0rth** [comment]
on a linux/amd64 host:
 CGO_ENABLED=1 GOOS=windows GOARCH=amd64 CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++   go build -ldflags -H=windowsgui awesomeUI.go 

generate a executable without a cmd window.

**Comment by andlabs** [comment]
Yes, I know that, but there is no way that I am aware of to add that flag automatically.
